mongo/session.go

```diff
@@ -307,6 +309,9 @@ func (s *sessionImpl) CommitTransaction(ctx context.Context) error {
 	}
 
 	err = op.Execute(ctx)
+	if IsTimeout(err) {
```

Comment:
Can you add a comment here explaining why we're ignoring timeout errors?

Reply:
Sure! And, rebased on master!